projects/pastanaga-angular/src/lib/controls/textfield/text-field.module.ts

```diff
@@ -8,10 +8,11 @@ import { PaDropdownModule } from '../../dropdown/dropdown.module';
 import { DeprecatedTextareaComponent } from './textarea/deprecated-textarea.component';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
 import { InputComponent } from './input/input.component';
+import { SelectComponent } from './select/select.component';
 
 @NgModule({
-    declarations: [InputComponent, DeprecatedSelectComponent, DeprecatedInputComponent, DeprecatedTextareaComponent],
+    declarations: [InputComponent, DeprecatedSelectComponent, DeprecatedInputComponent, DeprecatedTextareaComponent, SelectComponent],
```

Comment:
you should remove both `DeprecatedSelectComponent` and `DeprecatedInputComponent`. There is only one deprecated-select on home so you can have this PR in parallel and merge both together.

Reply:
as long as deprecatedInput is used in home, we can't... but that should not stay here for long

Reply:
that is my point: `deprecatedInput` is not used anywhere anymore, I did the replacement. The only place left was in the select and you just replaced it as well.
So you should remove both and replace `DeprecatedSelectComponent` from onboarding in a parallel PR.